lambdas/get-addresses/src/app.ts

```diff
@@ -1,34 +1,37 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
-
-/**
- *
- * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
- * @param {Object} event - API Gateway Lambda Proxy Input Format
- *
- * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
- * @returns {Object} object - API Gateway Lambda Proxy Output Format
- *
- */
+import { AddressService } from "./services/address-service";
+import config from "./config";
+import { DynamoDbClient } from "./lib/dynamo-db-client";
 
 export const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
     let response: APIGatewayProxyResult;
     try {
+        const sessionId = event.headers["session_id"] as string;
+
+        if (!sessionId) {
+            response = {
+                statusCode: 400,
+                body: "Missing header: session_id is required",
```

Comment:
Do we need this as this validation is already being done here https://github.com/alphagov/di-ipv-cri-address-api/pull/203/files#diff-027c6904962e08ce3c1694bc9a276b0644ba677533f3c15e0a6269d211c8ea03R184

Reply:
I added it for defensive programming purposes.

If we ever got rid of the APIGateway config or it didnt work as intended then there wouldn't be any check on the session ID check